doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -499,6 +499,10 @@ Bluetooth libraries and services
 
     * The :ref:`bt_mesh_le_pair_resp_readme` model to allow passing a passkey used in LE pairing over a mesh network.
 
+ :ref:`nrf_bt_scan_readme` library:
```

Comment:
```suggestion
 :ref:`nrf_bt_scan_readme`:
```
Not so sure about this. This library is called Scanning module. Feels a bit awkward to call it Scanning module library.